### DISCUSSION TITLE
New location for docs-l10n community discussions

### DISCUSSION THREAD

**Discussion by lamberta** [tensorflow/docs-l10n#376] [General]
We've consolidated communication channels for language communities to this GitHub Discussions board (in *beta*).

Please see the repo [README](https://github.com/tensorflow/docs-l10n/blob/master/README.md) for contributing instructions and use our [GitLocalize project](https://gitlocalize.com/tensorflow/docs-l10n) to submit pull requests and reviews: https://gitlocalize.com/tensorflow/docs-l10n

Issues can be filed under the *documentation* component of the [TensorFlow issue tracker](https://github.com/tensorflow/tensorflow/issues/new?template=20-documentation-issue.md). Questions about TensorFlow usage are better addressed on [Stack Overflow](https://stackoverflow.com/questions/tagged/tensorflow) or the [discuss@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/discuss) mailing list.

For now, we'll use a single discussion category for all languages but will explore multiple categories as new functionality is added to Discussions. And, as always, please abide by the [TensorFlow code of conduct](https://github.com/tensorflow/tensorflow/blob/master/CODE_OF_CONDUCT.md).

Welcome and enjoy!

**Comment by duynht** [comment]
Hi @lamberta 
I see that the `site/vi` directory is no longer there. Should I submit a PR to have it back?

**Reply by lamberta** [reply]
HI @duynht , 
No, we can leave as-is. `site/vi/` was removed in https://github.com/tensorflow/docs-l10n/pull/378, please see the comment from there:

> From the [README](https://github.com/tensorflow/docs-l10n/blob/master/README.md):
>
> Language support is determined by a number of factors including—but not limited to—site metrics and demand, community support, [English proficiency](https://en.wikipedia.org/wiki/EF_English_Proficiency_Index), audience preference, and other indicators. Since each supported language incurs a cost, unmaintained languages are removed. Support for new languages will be announced on the [TensorFlow blog](https://blog.tensorflow.org/) or [Twitter](https://twitter.com/TensorFlow).
>
> If your preferred language is not supported, you are welcome to maintain a community fork for open source contributors. These are not published to tensorflow.org.

Basically, we're going to be more deliberate about adding new languages to make sure the maintenance support is there. Thanks, though

**Reply by duynht** [reply]
I would love to host a community fork as Developer Student Clubs in Vietnam have just recruited more student volunteers. However, I couldn't find the repo of [https://www.tensorflow.org/](tensorflow.org) to mimic the CI/CD. Could you show me how, @lamberta ? 

Thank you.

**Reply by lamberta** [reply]
That sounds great, @duynht 
If you fork the the [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n) GitHub repo, the CI defined in [.github/workflows/ci.yaml](https://github.com/tensorflow/docs-l10n/blob/master/.github/workflows/ci.yaml) should work on your forked repo. But actions defined in the `bot-` files won't run since that requires our bot, which is unavailable.
In your fork, just create a `site/vi/` directory and mirror the file structure of `site/en-snapshot/`. You can even create your own https://gitlocalize.com project for your repo.
We won't publish this on tensorflow.org, but happy to add a link to your project once it's up and running—and it would benefit the Vietnamese community.

**Comment by KORshinjoonghyeok** [comment]
thanks for knowing me
